v-next.sql

```diff
@@ -24,8 +24,6 @@ CREATE TABLE openchpl.vendor_status_history (
 GRANT ALL ON TABLE openchpl.vendor_status_history TO openchpl;
 GRANT ALL ON SEQUENCE openchpl.vendor_status_history_vendor_status_history_id_seq TO openchpl;
 
-DROP TRIGGER vendor_status_history_audit on openchpl.vendor_status;
-DROP TRIGGER vendor_status_historys_timestamp on openchpl.vendor_status;
 CREATE TRIGGER vendor_status_history_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.vendor_status_history FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
```

Comment:
Gotcha. Hadn't thought about the table dropping triggering the dropped triggers, and the section of code showing here didn't show the "drop table" part which might have made me think of it